src/app/beer_garden/api/stomp/server.py

```diff
@@ -0,0 +1,65 @@
+import stomp
+from brewtils.models import Event, Events, Request, Operation, System
+from brewtils.schema_parser import SchemaParser
+import beer_garden.events
+import beer_garden.router
+from beer_garden.api.stomp.processors import append_headers, process_event_message
+
+conn = None
+
+
+class OperationListener(stomp.ConnectionListener):
+    def on_error(self, headers, message):
+        print('received an error:', headers)
+
+    def on_message(self, headers, message):
+        global conn
+        operation = SchemaParser.parse_operation(
+            message, from_string=True
+        )
+        if hasattr(operation, 'kwargs'):
+            if 'wait_timeout' in operation.kwargs and operation.operation_type == "REQUEST_CREATE":
+                operation.kwargs['wait_timeout'] = 0
+        result = beer_garden.router.route(operation)
+        if result is not None:
+            many = isinstance(result, list)
+            model_class = result.__class__.__name__
+
+            if many:
+                model_class = result[0].__class__.__name__
+            response_headers = {'model_class': model_class, 'many': many}
+            response_headers = append_headers(headers=response_headers)
```

Comment:
Can this have two inputs? 
```python 
response_headers = append_headers(response_headers=response_headers, request_headers=headers)
```

Then in the function we just don't do anything with the request_headers.